client/src/elm/Backend/Measurement/Model.elm

```diff
@@ -1016,6 +1017,14 @@ type TestResult
     | TestIndeterminate
 
 
+type BloodSmearResult
```

Comment:
@anvmn Do you think is worth to refactor this to something more specific:

```
type BloodSmearResult = Maybe BloodSmearTestResult

type BloodSmearTestResult = Negative 
                          | Positive NumberOfPlus
```

It seems a bit error prone to have everything inside one type. But up to you.

Reply:
@mariano-dagostino 
We list all possible results here, so, no need to refactor.
We have same structure ar numerous other places, and there were no issues with it.